### ISSUE TITLE
ReferenceError: Can't find variable: getinjectargs

### ISSUE THREAD

**Issue by Carniatto** [wardbell/bardjs#33]
Trying to use bardjs on my ionic project, when I run the tests I get:

ReferenceError: Can't find variable: getinjectargs
  at /Users/mcarniatto/Dev/xmobile/www/lib/bardjs/dist/bard.js:291

**Comment by ruslanss** [comment]
yes, this should only happen if you call inject without args, right?
